sources/packages/backend/libs/services/src/msfaa-number/msfaa-number-shared.service.ts

```diff
@@ -75,18 +75,21 @@ export class MSFAANumberSharedService {
       referenceApplicationId,
     );
     const pendingDisbursement =
-      application.currentAssessment.disbursementSchedules.find(
+      application.currentAssessment.disbursementSchedules.filter(
```

Comment:
As a suggestion we can just consider the first pending disbursement here like this. 
As if there are multiple pending disbursements they cannot have different msfaa's

```ts
const [pendingDisbursement] =
      application.currentAssessment.disbursementSchedules.filter(
        (schedule) =>
          schedule.disbursementScheduleStatus ===
          DisbursementScheduleStatus.Pending,
      );
    if (!pendingDisbursement) {
      throw new CustomNamedError(
        "Not possible to reissue an MSFAA when there is no pending disbursements for the application.",
        APPLICATION_INVALID_DATA_TO_CREATE_MSFAA_ERROR,
      );
    }
    if (!pendingDisbursement.msfaaNumber.cancelledDate) {
      throw new CustomNamedError(
        "Not possible to reissue an MSFAA when the current associated MSFAA is not cancelled.",
        APPLICATION_INVALID_DATA_TO_CREATE_MSFAA_ERROR,
      );
    }
```

Reply:
It makes sense, I will change it.

Reply:
Changed.